test/controls_column_test.dart

```diff
@@ -50,8 +50,8 @@ void main() {
       expect(closeButtonCallbackExecuted, true);
     });
 
-    testWidgets(' change navigation to drawing', (tester) async {
-      var mode = ControlMode.navigate;
+    testWidgets(' change drawing to navigating', (tester) async {
```

Comment:
Does this mean that the default mode is now the drawing mode? 
What made you change this? 
I'm not against this change, just curious why.

Reply:
No, the default mode is still navigating (see `defaultNavigate = true`). In this test, it is set to false.